.github/workflows/release.yml

```diff
@@ -13,7 +13,7 @@ jobs:
       uses: dawidd6/action-download-artifact@v2
       with:
         workflow: main.yml
-        branch: ${{ github.ref }}
+        commit: ${{ github.event.pull_request.head.sha }}
```

Comment:
Is this available for `workflow_dispatch` runs? I'd imagine `github.event.pull_request` will be `null` then. Maybe just use `github.sha`?

Reply:
Thank you for spotting that. I verified in the test branch and `github.sha` in a `workflow_dispatch` rightly points to the latest commit.